services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/services/director_v2/_thin_client.py

```diff
@@ -19,16 +19,18 @@
     retry_on_errors,
 )
 from servicelib.json_serialization import json_dumps
+from servicelib.rabbitmq.rpc_interfaces.dynamic_scheduler.services import (
+    DEFAULT_LEGACY_TIMEOUT_S,
+)
 
 from ...core.settings import ApplicationSettings
 
 
 class DirectorV2ThinClient(BaseThinClient, AttachLifespanMixin):
     def __init__(self, app: FastAPI) -> None:
         settings: ApplicationSettings = app.state.settings
-
         super().__init__(
-            request_timeout=10,
+            request_timeout=DEFAULT_LEGACY_TIMEOUT_S,
             base_url=settings.DYNAMIC_SCHEDULER_DIRECTOR_V2_SETTINGS.api_base_url,
             timeout=Timeout(10),
```

Comment:
And this hardocded timeout of 10? What i... what is the different between request_timeout and timeout parameters?